go/bump-vendored-golang/task

```diff
@@ -8,13 +7,25 @@ git config --global user.name "${GIT_USER_NAME}"
 GOLANG_RELEASE="$(pwd)/${GOLANG_DIR}"
 
 function make_private_yaml {
-  cat > config/private.yml <<EOF
+  if cat config/final.yml | grep "provider: gcs" > /dev/null ; then
+    cat > config/private.yml <<EOF
+---
+blobstore:
+  options:
+    credentials_source: static
+    json_key: |
+      $(echo ${GCS_JSON_KEY})
```

Comment:
Is the subshell to `echo` doing something clever here?

Reply:
lolz nope!